astrobee/behaviors/inspection/tools/inspection_tool.cc

```diff
@@ -538,24 +511,25 @@ int main(int argc, char *argv[]) {
   boost::thread inp(GetInput, &client);
 
   if (FLAGS_remote) {
-    ros::Rate loop_rate(10);
-    ros::Time start_time = ros::Time::now();
-
-    // Spin for 3 seconds
-    while (ros::Time::now() - start_time < ros::Duration(3.0))
-        loop_rate.sleep();
-
+    ros::Duration(3.0).sleep();
```

Comment:
I'll add a comment on this. When you declare a publisher you have to sleep a bit before publishing a message. This is not an issue when there is a connected_callback involved, but since we're doing remote commanding and waving that requirement, without this, the goal is not published. I can also lower the time involved, in retrospect 3s is too much..